mongod service is now rh-mongodb34-mongod.service for sat64

https://github.com/SatelliteQE/robottelo/blob/2561fef29258e9189cfbb83db595d1eab4c0e214/tests/foreman/installer/test_installer.py#L1159

```
2018-05-18 15:51:54 - robottelo.ssh - INFO - Connected to [qe-sat64-rhel7.satqe.lab.eng.rdu2.redhat.com]
2018-05-18 15:51:54 - robottelo.ssh - INFO - >>> systemctl status mongod
2018-05-18 15:51:55 - robottelo.ssh - INFO - <<< stderr
Unit mongod.service could not be found.
```

```
        for service in services:
            with self.subTest(service):
                result = ssh.command(status_format.format(service))
>               self.assertEqual(result.return_code, 0)
E               AssertionError: 4 != 0
```

--- a/tests/foreman/installer/test_installer.py
+++ b/tests/foreman/installer/test_installer.py
@@ -1156,7 +1156,7 @@ class SELinuxTestCase(TestCase):
         :id: 85fd4388-6d94-42f5-bed2-24be38e9f104
 
         :expectedresults: All services {'elasticsearch', 'foreman-proxy',
-            'foreman-tasks', 'httpd', 'mongod', 'postgresql',
+            'foreman-tasks', 'httpd', 'rh-mongodb34-mongod', 'postgresql',
             'pulp_celerybeat', 'pulp_resource_manager', 'pulp_workers',
             'qdrouterd', 'qpidd', 'tomcat'} are started
         """
@@ -1165,7 +1165,7 @@ class SELinuxTestCase(TestCase):
             'foreman-proxy',
             'foreman-tasks',
             'httpd',
-            'mongod',
+            'rh-mongodb34-mongod',
             'postgresql',
             'pulp_celerybeat',
             'pulp_resource_manager',

